src/cmd_start.rs

```diff
@@ -310,6 +310,13 @@ pub fn command_start< 'a >( matches: &clap::ArgMatches< 'a > ) -> Result< (), Er
     eprintln!( "" );
     eprintln!( "You can access the web server at `http://{}`.", &address );
 
+    if matches.is_present( "open" ) {
+        thread::spawn( move || {
+            thread::sleep( Duration::from_millis( 100 ) );
```

Comment:
Hmm... ideally we'd want to actually wait for the server to start instead of this bare sleep, although I'm not really sure how to do it cleanly.

Unless you have a better idea on how to do this for now could you add a busy loop, e.g. something like this with some slight modification: (copied verbatim from `integration-tests/src/main.rs`):

```
        let start = Instant::now();
        let mut response = None;
        while start.elapsed() < Duration::from_secs( 10 ) && response.is_none() {
            thread::sleep( Duration::from_millis( 100 ) );
            response = reqwest::get( "http://localhost:8000" ).ok();
        }
```

For URL we can probably use `/__cargo-web__/build_hash` as something that should always be GETable.